packages/changed-elements-react/src/NamedVersionSelector/NamedVersionSelector.tsx

```diff
@@ -647,25 +652,25 @@ function ProcessingEntryStatus(props: ProcessingEntryStatusProps): ReactElement
       <ProgressRadial size="x-small" data-progress={progress} value={progress} >
       </ProgressRadial>
     </Flex> :
-    <Flex>
+      <Flex>
```

Comment:
Nitpick: To make this block more readable maybe consider:
```
return (props.displayMin
  ? <Flex>
  ...
  : <Flex>
  ...
```

or better yet, just put this in an if statement before
```
if (props.displayMin) {
    return (<Flex> .... </Flex>);
}

return <Flex> ....;
```

Should these states also be their own simple functional components based on the progress variable? since that seems like the only input to them?